src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
 // write your code here
+// Отримуємо всі елементи <span> з класом "population"
+const populationSpans = document.querySelectorAll('span.population');
+// Масив для збереження числових значень
+const populationValues = [];
+
+// Проходимо через кожен елемент, перевіряємо та конвертуємо текст у число
+populationSpans.forEach((span) => {
+  const populationText = span.textContent.replace(/,/g, '');
+  const populationNumber = parseFloat(populationText);
+
+  // Перевіряємо, чи є значення числом
+  if (!isNaN(populationNumber)) {
+    populationValues.push(populationNumber); // Додаємо в масив, якщо це число
+  }
+});
+
+// Обчислюємо загальну кількість
+const totalPopulation = populationValues.reduce((sum, value) => sum + value, 0);
+
+// Обчислюємо середнє значення
+const averagePopulation = totalPopulation / populationValues.length;
+
+// Функція для форматування чисел з тисячним розділювачем
+function formatNumberWithCommas(number) {
+  return number.toLocaleString();
```

Comment:
You are using the toLocaleString method to format numbers with commas. This is a good approach, but keep in mind that the output will depend on the user's locale. For example, in some locales, the comma is used as a decimal separator. If you want to ensure that the number is always formatted with commas as thousands separators, you could use a more specific locale, like 'en-US'.